### DISCUSSION TITLE
NBMiner v41.3

### DISCUSSION THREAD

**Discussion by nebutech-admin** [NebuTech/NBMiner#851] [General]
- `fix`: `ethash` Improve stability of LHR unlocker.
- `fix`: `ethash` Fix crash on AMD GPUs
- `fix`: `ethash` Improve compatibility on rigs with small system memory.
- `note`: **Recommend driver versions: `512.15` for Windows, `510.60` for Linux.**

<hr /><em>This discussion was created from the release <a href='https://github.com/NebuTech/NBMiner/releases/tag/v41.3'>NBMiner v41.3</a>.</em>

**Comment by frostmagnetic** [comment]
Please, how to find installation script for Linux?

**Reply by razakhan** [reply]
If you already have 41.0 running 
cd /tmp && wget https://dl.nbminer.com/NBMiner_41.3_Linux.tgz && tar -xvf NBMiner_41.3_Linux.tgz && cd NBMiner_Linux && miner stop && cp nbminer /hive/miners/nbminer/41.0 && miner start

**Comment by ABKartashov** [comment]


**Reply by razakhan** [reply]
Remove the V1 RTX 3060 that is no supported i think

**Reply by ABKartashov** [reply]
Thank you

**Reply by bierschurz** [reply]
with 41.0 3060 v1 works flawless. at least with w10.
have not test new version yet.

**Comment by mhairt** [comment]
When I try and run as admin I get this message
<img width="638" alt="nbminer" src="https://user-images.githubusercontent.com/69930860/167607052-e4a192df-29fa-41bc-bcba-0eca97ae379f.png">

**Reply by WygorFonseca** [reply]
Are you trying to run as administrator by opening cmd as administrator? Or by right clicking on the .bat and going to "Run as administrator"?

**Comment by rizpardazan** [comment]
more stable that 41.0
work with lhr v1 and v2

**Comment by Teddymeyn** [comment]
Edit the sample Bat file with your configuration n then run that as admin.

On Tue, 10 May 2022 at 3:49 PM mhairt ***@***.***> wrote:

> When I try and run as admin I get this message
> [image: nbminer]
> <https://user-images.githubusercontent.com/69930860/167607052-e4a192df-29fa-41bc-bcba-0eca97ae379f.png>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NebuTech/NBMiner/discussions/851#discussioncomment-2721212>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AVJEXUVYXXTD6YUHA2NDUJTVJIZ4TANCNFSM5VQPWCUA>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>
-- 
Sent from Gmail Mobile

**Comment by Danishagn** [comment]
Still problem with 3060ti and 3070ti
Using HIVEOS LATEST VERSION
Driver 510.68.02 and 510.60 BOTH VERSION
0% FAN AND HALF HASHRATE FROM FULL HASH

3060TI OC 1485, 2800, 140W, FAN 80%
3070TI OC  900, 2300, 200W, FAN 80%

AFTER FEW HOURS RUNNING this happen with 1-2 cards some time 3 cards also

**Reply by caramelzane** [reply]
Same issue for 3060 V2.

Stability issues similar to NBMiner v41.0

**Reply by odinsy** [reply]
@Danishagn yesterday I've run couple 3060TI LHR with 510.60.02 driver and OC 1350, 1600, 150W - still works stably with 58 MH/s

**Reply by Danishagn** [reply]
I have problem in some rigs and some are working fine... We have rigs like 3060TI 11 gpu in one rig and 3070ti 11 gpu in one rig, less qty of gpu of same model in one rig wont giving any problem

**Comment by Esla91** [comment]
<a href="https://ibb.co/LdwWyKx"><img src="https://i.ibb.co/LdwWyKx/279919797-3035719940042476-6441383465556840813-n.jpg" alt="279919797-3035719940042476-6441383465556840813-n" border="0"></a>

and the rig close :(

**Reply by nbminer-beta** [reply]
This issue is about your pool not for miner.

You should try DNS or if you using dns try another dns its seem dns/connection problem.

Also I suggest you try another pool to sure its mining better or not.

Always try if its safe with -100,-200 mclcok ratio.

**Reply by Esla91** [reply]
<a href="https://ibb.co/HXvKD81"><img src="https://i.ibb.co/HXvKD81/280470568-378044290926368-1476198057241026497-n.jpg" alt="280470568-378044290926368-1476198057241026497-n" border="0"></a>

changed the dns twice and the mclock but the rig restart too, the problem in ethermine pool now right ?

**Reply by Danishagn** [reply]
Ethermine pool block its services in some countries maybe you are in the list ... where ethermine blocks

**Reply by Esla91** [reply]
but I haven't any problem with other miner with the same pool

**Reply by Danishagn** [reply]
Please check internet connection and oc setting, lower the memory setting also
Unplug your internet cable and plug in and reboot the rig or reboot the router maybe internet issue some time happen on same router same network one rig working and one rig showing that internet is offline

**Reply by Esla91** [reply]
<a href="https://ibb.co/nk59S9H"><img src="https://i.ibb.co/nk59S9H/280559823-418253353459290-7988588862539130223-n.jpg" alt="280559823-418253353459290-7988588862539130223-n" border="0"></a> <a href="https://ibb.co/WpGK8wj"><img src="https://i.ibb.co/WpGK8wj/280253310-304396645216074-4319826744244715297-n.jpg" alt="280253310-304396645216074-4319826744244715297-n" border="0"></a> <a href="https://ibb.co/HHyBzkY"><img src="https://i.ibb.co/HHyBzkY/279963587-3149910545323073-4740008002142065035-n.jpg" alt="279963587-3149910545323073-4740008002142065035-n" border="0"></a>

same problem, the only thing i didnt try is changing the pool

**Reply by Danishagn** [reply]
Host not found error mostly internet issue... you have other rigs connected with same network and same pool?

**Reply by Esla91** [reply]
> Host not found error mostly internet issue... you have other rigs connected with same network and same pool?

unfortunately, I don't have it, but I used another miner, and works fine now
thank you and many thanks to NBminer

**Comment by AthaMerth** [comment]
was getting 36Hashrate with RTX 3060 LHRV2 12GB now getting 46Hashrate. Good Work!

**Comment by username-rand** [comment]
The developers are miracle workers.  I was running a EVGA 3080 12GB with another LHR unlock miner and it was running about 64MH/s.  I switched to nbminer 41.3 and the 3080 12GB ran about 74 MH/s.  Today I switched to 41.4 and the 3080 12GB ran at 101 MH/s so far it has been stable for about an hour.  Many thanks to the developers.

**Comment by username-rand** [comment]
I found it here

https://www.reddit.com/r/EtherMining/comments/umjz8s/beta_nbminer_414_90_lhr_v3_unlock/ <https://www.reddit.com/r/EtherMining/comments/umjz8s/beta_nbminer_414_90_lhr_v3_unlock/>

**Reply by username-rand** [reply]
nbminer 41.4 beta now appears to be available to select using HiveOS (not latest, you have to select that particular version)

**Comment by Danny87Chew** [comment]
I use 41.0 on my RTX 3070 LHR with 460.91.03 driver installed, then the hashrate actually dropped a lot(27M) and infomation not displayed correctly as well. When I use 40.1, same setting I can get 44M
Version 41.0
![image](https://user-images.githubusercontent.com/66708807/167769049-5cb74c89-ccbf-4b0d-a439-934e5930df26.png)
Version 40.1:
![image](https://user-images.githubusercontent.com/66708807/167769399-8dd21622-d440-44e1-8e9a-8a01292f1f29.png)

**Reply by Danishagn** [reply]
Use NBminer 41.3 and Driver 510.60 or 510.68 in hiveos if you are in windows then use latest nvidia drivers 512
in hiveos edit flight sheet and goto miner config options select version manually 41.3 then update it

**Comment by s0laria** [comment]
13 hours without errors. Perfect!

**Comment by cybeattack** [comment]
We need more version admin. Fix crash GPU. Now not stable 100% some cards have crash.

**Comment by lenxiety** [comment]
It works on 512.59 or must revert driver version? thanks
